### DISCUSSION TITLE
external log file to reduce memory usage

### DISCUSSION THREAD

**Discussion by GravityRZ** [koush/scrypted#682] [Q&A] [answered]
is there a way to put the log file on an external location instead of inside the container.
i notice the container is using a lot of memory and it is increasing rapidly

**Comment by koush** [comment]
log management can be handled with your docker compose configuration. set the logging driver to none.

**Comment by GravityRZ** [comment, marked as answer]
and how can i do that without using docker compose. i hav send up the container the manual way(synology/docker)

**Reply by GravityRZ** [reply]
ok. solved it by starting the container from the commandline. docker gui on synology is very limited

**Comment by johntdyer** [comment]
What about running it on OSX as a standalone application ?
